diff --git a/wp-content/plugins/addon-so-widgets-bundle/cmb/init.php b/wp-content/plugins/addon-so-widgets-bundle/cmb/init.php
index 742ca5ff..497b5130 100644
--- a/wp-content/plugins/addon-so-widgets-bundle/cmb/init.php
+++ b/wp-content/plugins/addon-so-widgets-bundle/cmb/init.php
@@ -743,7 +743,7 @@ class cmb_Meta_Box {
 		elseif ( is_string( $meta_box['pages'] ) )
 			$type = $meta_box['pages'];
 		// if it's an array of one, extract it
-		elseif ( is_array( $meta_box['pages'] ) && count( $meta_box['pages'] === 1 ) )
+		elseif ( is_array( $meta_box['pages'] ) && count( $meta_box['pages'] ) === 1 )
 			$type = is_string( end( $meta_box['pages'] ) ) ? end( $meta_box['pages'] ) : false;
 
 		if ( !$type )
